spec/datadog/core/diagnostics/health_spec.rb

```diff
@@ -3,9 +3,12 @@
 require 'spec_helper'
 require 'ddtrace'
 require 'datadog/core/diagnostics/health'
+require 'datadog/statsd'
 
 RSpec.describe Datadog::Core::Diagnostics::Health::Metrics do
```

Comment:
Should this file being rename to `spec/datadog/core/diagnostics/health/metrics_spec.rb`?